opm/simulators/flow/FlowGenericProblem_impl.hpp

```diff
@@ -762,8 +750,7 @@ operator==(const FlowGenericProblem& rhs) const
            this->solventSaturation_ == rhs.solventSaturation_ &&
            this->solventRsw_ == rhs.solventRsw_ &&
            this->polymer_ == rhs.polymer_ &&
-           this->micp_ == rhs.micp_ &&
-           this->mixControls_ == rhs.mixControls_;
+           this->micp_ == rhs.micp_;
```

Comment:
I did not figure out whether FlowProblem needs a `operator==`, at least, it is not there yet.

Reply:
Again as there is no test there is no need for the comparison operator.